src/Validator/Rules/QueryDepth.php

```diff
@@ -15,6 +15,9 @@
 
 class QueryDepth extends QuerySecurityRule
 {
+    /** @var array<string, bool> hash of fragment names which are already calculated in recursion */
```

Comment:
```suggestion
    /** @var array<string, bool> Fragment names which are already calculated in recursion */
```

I guess you are referring to hash as in the datastructure HashMap, but there is no such thing in PHP. I assumed to find a usage of a hashing function somewhere in the code, but this is just a plain associative array.

Reply:
Yep, I meant "[Hash table](https://en.wikipedia.org/wiki/Hash_table)", not to confuse with just `array<string>`